packages/generator/src/GenerateContext/EnhancedAction.ts

```diff
@@ -36,7 +36,7 @@ export class EnhancedAction extends EnhancedBase<ActionTypeV2> {
   }
 
   get paramsIdentifier() {
-    return `${this.shortApiName}.Parameters`;
+    return `${this.shortApiName}.Params`;
```

Comment:
Fine with the rename, but just wondering why?

Reply:
Got annoyed in the internal repo typing out the long string. Imagine if we used `Properties` instead of `Props` for all of our react components?